api/server/volume_test.go

```diff
@@ -2415,37 +2415,6 @@ func TestMiddlewareVolumeDeleteSuccess(t *testing.T) {
 
 }
 
-func TestMiddlewareVolumeDeleteFailure(t *testing.T) {
```

Comment:
This test is no longer applicable. It was failing because a token was not provided.